.devcontainer/Dockerfile

```diff
@@ -77,6 +78,7 @@ fi\n\
  \n\
 alias ll="ls -alF"\n\
 alias ls="ls --color=auto"\n\
+export GATEWAY_IP=$(ip route | awk "/default/ { print \$3 }") \

```

Comment:
I'm not sure that `GATEWAY_IP` is the right name for this.  Granting access to the default gateway is wrong in general. It only works because in this particular setup the default gateway is the same as the host.

Reply:
Will change it in a future PR. host.docker.internal however resolves to an IP that is not the same as what we were seeing